### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MiniMixer**
[DEMO LINK] (https://minimixer.github.io/react_todo-app-add-and-delete/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,105 @@
...
+      });
+  };
+
+  const deleteCompleted = () => {
+    const completedNum = todosFromServer.filter(todo => todo.completed);
+
+    for (let i = 0; i < completedNum.length; i++) {
+      deleteTodo(completedNum[i].id);
+    }
+  };
```
Comment (etojeDenys) [inline src/App.tsx:73]: it would be better to use promise.all() here

**Review by etojeDenys — CHANGES_REQUESTED** [review]
good job, to improve:

after trying to delete several todos in a short period of time, the loader only appears on the last one pressed

**Review by brespect — CHANGES_REQUESTED** [review]
Seems like, comment from previous mentor`s review was not fixed

**Comment by MiniMixer** [comment]
> Seems like, comment from previous mentor`s review was not fixed

My bad. I just forgot to deploy new version

File: src/components/TodoItem/TodoItem.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+      >
+        <div className="modal-background has-background-white-ter" />
+        <div className="loader" />
+      </div>
+    </div>
+  );
+};
+
+{
+  /* <div data-cy="Todo" className="todo completed">
```
Comment (brespect) [inline src/components/TodoItem/TodoItem.tsx]: remove all unused code and comments

**Review by brespect — CHANGES_REQUESTED** [review]
Still didn`t see **Promise.all** realization

File: src/App.tsx (hunk 1 above)
Comment (Anton-Kuchmasov) [reply src/App.tsx:73]: nice!

**Review by Anton-Kuchmasov — APPROVED** [review]
Well done!

Try to implement `Promise.allSettled()` in the next chapter to handle errored todos in edge cases (should be checked by tests)
